thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -130,4 +130,19 @@ if (!(Test-Path "Eluant.dll"))
 	(New-Object System.Net.WebClient).DownloadFile("https://github.com/OpenRA/Eluant/releases/download/20140425/Eluant.dll", $target)
 }
 
+if ((Test-Path "GeoLite2-Country.mmdb"))
+{
+	$lastWrite = (get-item "GeoLite2-Country.mmdb").LastWriteTime
+	$timespan = new-timespan -days 30
+}
+
+if (!(Test-Path "GeoLite2-Country.mmdb") -Or (((get-date) - $lastWrite) -gt $timespan))
+{
+	echo "Updating GeoIP country database from MaxMind."
+	$target = Join-Path $pwd.ToString() "GeoLite2-Country.mmdb.gz"
+	(New-Object System.Net.WebClient).DownloadFile("http://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.mmdb.gz", $target)
+	7z e GeoLite2-Country.mmdb.gz
```

Comment:
I don't know any other way on Windows. According to AppVeyor staff this is installed and available in PATH.